src/main/java/graphql/annotations/EnhancedExecutionStrategy.java

```diff
@@ -55,20 +63,45 @@ protected ExecutionResult resolveField(ExecutionContext executionContext, GraphQ
                 clientMutationId = clientMutationIdVal.getValue();
             }
 
-            return completeValue(executionContext, fieldDef.getType(), fields, clientMutationId);
+            TypeInfo fieldTypeInfo = newTypeInfo(fieldDef.getType(), parameters.typeInfo());
+            ExecutionParameters newParameters = newParameters()
```

Comment:
thanks.  The next version of graphql-java is also a breaking change 4.x

We will re-release then